packages/ui/src/theme/tokens/components/alert.js

```diff
@@ -1,23 +1,37 @@
 module.exports = {
   // Default styles
+  color: { value: '{colors.font.primary.value}' },
   backgroundColor: { value: '{colors.background.tertiary.value}' },
-  paddingVertical: { value: '{space.small.value}' },
-  paddingHorizontal: { value: '{space.medium.value}' },
+  paddingBlock: { value: '{space.small.value}' },
```

Comment:
Changing these to logical names: https://developer.mozilla.org/en-US/docs/Web/CSS/padding-block